kubernetes-docs/overview.md

```diff
@@ -0,0 +1,49 @@
+---
+title: vSphere Cloud Provider
+---
+
+## Overview
+Containers are stateless and ephemeral but applications are stateful and need persistent storage. vSphere adds this persistent storage support to Kubernetes through interface called Cloud Provider. Cloud provider is an interface which helps in extending Kubernetes with cluster of instances managed by virtualisation technologies, public/private cloud platforms and required networking for these instances. 
+ 
+ ![Image](images/vSphere.png)
```

Comment:
I have updated it to increase by 30%. Beyond this we might need a high resolution image to scale.